### PR TITLE
Update usage.md

### PR DESCRIPTION
**Pull request by lucy-kind**
Update usage instructions with .jar file asset download link.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The tool is a command line application built with Java. See below for usage inst
 
 # Usage
```
Comment (juliawu) [inline docs/usage.md:17]: Perhaps we should move the section you just added to a new 'Installation' section.

File: docs/usage.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The tool is a command line application built with Java. See below for usage inst
...
 
+Make sure you've downloaded the .jar file under Assets [here](https://github.com/datacommonsorg/import/releases/tag/0.1-alpha.1h). Note the path to .jar.
```
Comment (juliawu) [inline docs/usage.md:19]: Let's link to just the releases page (https://github.com/datacommonsorg/import/releases) incase we ever release newer versions. We can ask users to download the latest.

**Review by juliawu — CHANGES_REQUESTED** [review]
Thanks for the much needed update!

File: docs/usage.md (hunk 1 above)
Comment (lucy-kind) [reply docs/usage.md]: Done! ptal

**Review by juliawu — APPROVED** [review]
Thanks for the changes!

**Pull request merged**